libcontainer/cgroups/fscommon/rdma.go

```diff
@@ -0,0 +1,123 @@
+// +build linux
+
+package fscommon
+
+import (
+	"bufio"
+	"errors"
+	"math"
+	"strconv"
+	"strings"
+
+	"github.com/opencontainers/runc/libcontainer/cgroups"
+	"github.com/opencontainers/runc/libcontainer/configs"
+	"golang.org/x/sys/unix"
+)
+
+// parseRdmaKV: parses raw string to RdmaEntry.
+func parseRdmaKV(raw string, entry *cgroups.RdmaEntry) error {
+	var value uint64
```

Comment:
As you are casting the value to uint32 below, maybe it makes sense to have it as uint32.

```go
	var value uint32
...
	if v == "max" {
		value = math.MaxUint32
	} else {
		val64, err := strconv.ParseUint(v, 10, 32)
		if err != nil {
			return err
		}
		value  = uint32(val64)
	}
```

Reply:
@kolyshkin Resolved thanks for spotting this.